From ec42dedce2085779e8477ef9bc1aa2d3231f4a28 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Mon, 30 Nov 2020 15:37:20 +0200
Subject: [PATCH] [SW-1833] preboot: add aligned macro

Add aligned macro to be used for forcing alignment on structures

Change-Id: I364f428644381f963698eeafb65eb5183587b922
Signed-off-by: Guy Nisan <gnisan@habana.ai>
(cherry picked from commit 8721827bc55fd8516dedc23171ca143d08e239cd)
---

diff --git a/include/preboot.h b/include/preboot.h
index 47f6047..db9532d 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -52,6 +52,9 @@
 #ifndef __weak
 #define __weak		__attribute__((weak))
 #endif
+#ifndef __aligned
+#define __aligned(x)	__attribute__((aligned(x)))
+#endif
 
 typedef unsigned char u8;
 typedef unsigned int u32;
